lib/matplotlib/testing/__init__.py

```diff
@@ -1,19 +1,15 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+import six
```

Comment:
Not needed in this file any more.

Reply:
They are all going away soon anyway so :man_shrugging: